src/settings.js

```diff
@@ -1292,39 +1283,56 @@ var ASMFS = 0;
 // then you can safely ignore this warning.
 var SINGLE_FILE = 0;
 
-// For internal use only (name of the file containing wasm text, if relevant).
+// if set to 1, then generated WASM files will contain a custom
+// "emscripten_metadata" section that contains information necessary
+// to execute the file without the accompanying JS file.
+var EMIT_EMSCRIPTEN_METADATA = 0;
```

Comment:
I think you added this setting my mistake here.  It was already added in #7815

Reply:
> I think you added this setting my mistake here. It was already added in #7815

Sorry, I don't understand what you mean here?

Reply:
EMIT_EMSCRIPTEN_METADATA now appears twice in this file, after this change I think..

Reply:
Oh, thanks, must be a bad merge. I can clean that up unless you were already looking at stuff in that file anyhow and planned to?

Reply:
I don't have a specific CL that that no